create_infotexts.py

```diff
@@ -168,24 +172,25 @@ def create_smvk_mm_link(item):
 
 
 def select_best_mapping_for_depicted_person(flipped_name):
+    """Lookup available mappings for a string respresenting a name and return best choice.
+
+    :flipped_name: string representing full name turned around from e.g. "surname, given name" -> "given name surename"
+    :return:  maps a name to it's best mapping; wikidata, then commons and lastly the name only
```

Comment:
This describes what the function does, not the return value.

I would move the selection order to the general docstring and here saysomething like "string used  to best map name" or something.